dcc-portal-ui/app/scripts/ui/js/table.js

```diff
@@ -238,6 +244,24 @@ angular.module('icgc.ui.table.pagination', [])
         };
       }
     };
+  })
+  .directive('paginationClientSide', function(){
```

Comment:
I actually thought of combining them into one but I am not sure how that would workout. Both the directives have different functionalities. For server side pagination, we only request data based on page number and row limit. However for client side pagination, we already have all the data available we sort of restrict front end to only show based on user selection. Also, server side pagination depends to FilterService while client side pagination has no dependency.